modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -77,27 +77,27 @@ The `--copy-network` option only copies networking configuration found under `/e
 
 The disk partitions are created on {product-title} cluster nodes during the {op-system-first} installation. Each {op-system} node of a particular architecture uses the same partition layout, unless the default partitioning configuration is overridden. During the {op-system} installation, the size of the root file system is increased to use the remaining available space on the target device.
 
+Kubernetes supports monitoring `/var/lib/containers` when mounted in the same partition as `/var` or a separate partition. 
+
 There are two cases where you might want to override the default partitioning when installing {op-system} on an {product-title} cluster node:
 
-* Creating separate partitions: For greenfield installations on an empty
-disk, you might want to add separate storage to a partition. This is
-officially supported for mounting `/var` or a subdirectory of `/var`, such as `/var/lib/etcd`, on a separate partition, but not both.
+* Creating separate partitions: For greenfield installations on an empty disk, you might want to add separate storage to a partition. For example, mount `/var/lib/containers` in a separate partition, so that kubelet can monitor the `imagefs` directory for container runtimes,`/var/lib/containers` and the `nodefs` directory for root directory files.
```

Comment:
The above is confusing, better:

```
* Creating separate partitions: For greenfield installations on an empty disk, you might want to add a separate storage partition for your containers and container images. For example, by mounting `/var/lib/containers` in a separate partition, the kubelet will separately monitor `/var/lib/containers` as the `imagefs`. And it will monitor the root file system as the `nodefs`.
```

Reply:
Agree, the original phrasing is confusing and the one suggested by @andreaskaris is better.  
I would in addition drop the part about greenfield and empty disk. When do we ever deploy OpenShift to a Node which does not have an empty disk? Do if we even support such a model, and what would this disk then contain?

Reply:
Thanks, @andreaskaris and @djuran. How about:

> Creating separate partitions: You might want to add a separate storage partition for your containers and container images. For example, by mounting `/var/lib/containers` in a separate partition, the kubelet will separately monitor `/var/lib/containers` as the `imagefs` directory and the root file system as the `nodefs` directory.

Reply:
Thanks, @djuran . Removed. I moved the following IMPORTANT note closer to the intro paragraph: 

![Screenshot from 2024-03-12 10-50-54](https://github.com/openshift/openshift-docs/assets/57954076/3886faec-febe-44ad-9b10-7cd28bd6eb25)